CODEOWNERS

```diff
@@ -33,27 +53,27 @@
 /tests/unit/test_docker_utils.py @dfangl @dominikschubert
 /tests/unit/test_dockerclient.py @dfangl @dominikschubert
 
+# Package Installers
+/localstack/packages/ @alexrashed
+/localstack/services/kinesis/packages.py @alexrashed
+
 # DNS server
 /localstack/dns @simonrw @dfangl
 
 # HTTP framework
 /localstack/http/ @thrau
 /tests/unit/http_/ @thrau
 
+# Runtime
+/localstack/runtime/ @thrau
+
+# Logging
+/localstack/logging/ @thrau @alexrashed @dominikschubert
```

Comment:
i'm happy to hand off code ownership of logging. @dfangl did a lot of the initial implementation